packages/launcher-component/src/pickers/buildimage-picker.tsx

```diff
@@ -16,37 +18,55 @@ interface BuildImageProps extends InputProps<BuildImageValue> {
 
 export function BuildImagePicker(props: BuildImageProps) {
   if (!props.value.imageName) {
-    props.onChange({imageName: props.result.image});
+    props.onChange({ imageName: props.result.image });
     return (<Loader />);
   }
-  const imageName = props.value.imageName;
   return (
     <Fragment>
-      <p>Analyzed your code, this is the build image we suggest: <i>"{imageName}"</i></p>
-      <p>But we could be wrong, you can change the image here:</p>
-      <Form>
-        <FormGroup
-          label="Build image"
-          isRequired
-          fieldId="imageName"
-        >
-          <FormSelect
-            id="imageName"
-            value={imageName}
-            onChange={value => props.onChange({imageName: value})}
-            aria-label="Select build image"
-          >
-            {props.result.builderImages.map((image, index) => (
-              <FormSelectOption
-                key={index}
-                value={image.id}
-                label={image.name}
-              />
-            ))
+      <p>Analyzed your code, this is the build image we suggest: <i>"{props.result.image}"</i></p>
```

Comment:
@edewit You can use `<SpecialValue>{props.result.image}</SpecialValue>` to make it more obvious.

Instead of `Analyzed your code, , this is the build image we suggest:`, let's use `For your codebase, our runtime detection algorithm suggest this image: <SpecialValue>{props.result.image}</SpecialValue>`